.github/PULL_REQUEST_TEMPLATE/bug_fix.md

```
@@ -0,0 +1,47 @@
+assignees: ''
+---
+
+**App Name:** (e.g., `web`, `backend`, `shared-types`, `ui`, `monorepo-root`)
+
+**Description**
+Please include a summary of the bug being fixed and the changes made to resolve it. Also, include any relevant context.
+
+**Related Issue**
+Fixes # (issue number)
+
+**Type of change**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)